Pod/Classes/ViewControllers/ARMapComponentViewController.m

```diff
@@ -4,7 +4,14 @@ @implementation ARMapComponentViewController
 
 - (instancetype)init
 {
-    return [super initWithEmission:nil moduleName:@"Map" initialProperties:nil];
+    return [super initWithEmission:nil moduleName:@"Map" initialProperties:@{
+        @"safeAreaInsets": @{
+            @"top": @(0),
+            @"left": @(0),
+            @"bottom": @(0),
+            @"right": @(0)
+        }
+    }];
```

Comment:
Maybe this can live inside the VC - inside an else on this func: https://github.com/artsy/emission/pull/1434/files#diff-a6663cfe9d0e55272a71b62f28e85090R234

Reply:
I'd prefer to have it from the get-go. The `Props` define it as a required property and this decouples the React component's requirements from our ObjC code (what if `updateSafeAreaInsets` isn't called by the time the first render occurs?).

Reply:
👍

Reply:
But I can see it either way, since `updateSafeAreaInsets` looks pretty critical to the UI anyway. Yeah, let's do that so our support for older devices is explicit 👍